local_publisher.py

```diff
@@ -22,25 +22,39 @@ class LocalPublisher(TerminatorBlock):
 
```

Comment:
On line 19, does `topic` need to be a `Property` or `ExpressionProperty` rather than a `StringProperty`?